README.md

```diff
@@ -89,7 +99,7 @@ window.onload = ->
   return
 ```
 
-or create `app/assets/javascripts/home.js` and insert
+If you don't want to use CoffeeScript, delete `app/assets/javascripts/home.coffee`, create `app/assets/javascripts/home.js` and insert
 
 ```javascript 
 window.onload = function () {
```

Comment:
Can you update the `apiKey` variable and comment to match the explanation in the React demo? We need to make clear, that the SDK doesn't require API-Keys, but the contents of the license files.